opentracing/span.py

```diff
@@ -70,28 +70,15 @@ def set_operation_name(self, operation_name):
         """
         return self
 
-    def start_child(self, operation_name, tags=None):
-        """Denotes the beginning of a subordinate unit of work.
-
-        As this is a no-op implementation, it actually returns itself as the
-        child span. Real implementations should create a new span.
-
-        :param operation_name: name of the operation represented by the new
-            span from the perspective of the current service.
-        :param tags: optional dictionary of Span Tags. The caller is
-            expected to give up ownership of that dictionary, because the
-            Tracer may use it as is to avoid extra data copying.
-
-        :return: Returns a new child Span in "started" state.
-        """
-        return self
-
-    def finish(self):
+    def finish(self, **kwargs):
         """Indicates that the work represented by this span has been completed
         or terminated, and is ready to be sent to the Reporter.
 
         If any tags / logs need to be added to the span, it should be done
         before calling finish(), otherwise they may be ignored.
+
+        :param finish_time: an explicit Span finish timestamp as a unix
```

Comment:
I would prefer to list optional arguments explicitly with appropriate defaults instead of using `kwargs`, it allows compiler to do stronger checks.